spec/asciidoctor/rfc/v3/author_spec.rb

```diff
@@ -15,7 +15,7 @@
       <seriesInfo name="Internet-Draft" stream="IETF" value=""/>
       <seriesInfo name="" value=""/>
 
-      <author fullname="John Doe Horton" initials="J" surname="Horton">
```

Comment:
@paolobrasolin the "initials" attribute is now missing from the specs. Were these intentional?

Reply:
I had moved the input header attribute from initials to forename_initials, and the spec wasn't catching that. Restoring.